src/sentry/models/outbox.py

```diff
@@ -51,13 +51,14 @@ class OutboxCategory(IntEnum):
     WEBHOOK_PROXY = 1
     ORGANIZATION_UPDATE = 2
     ORGANIZATION_MEMBER_UPDATE = 3
-    VERIFY_ORGANIZATION_MAPPING = 4
-    AUDIT_LOG_EVENT = 5
-    USER_IP_EVENT = 6
-    INTEGRATION_UPDATE = 7
-    PROJECT_UPDATE = 8
-    API_APPLICATION_UPDATE = 9
-    SENTRY_APP_INSTALLATION_UPDATE = 10
+    ORGANIZATION_MEMBER_CREATE = 4
+    VERIFY_ORGANIZATION_MAPPING = 5
+    AUDIT_LOG_EVENT = 6
+    USER_IP_EVENT = 7
+    INTEGRATION_UPDATE = 8
+    PROJECT_UPDATE = 9
+    API_APPLICATION_UPDATE = 10
+    SENTRY_APP_INSTALLATION_UPDATE = 11
```

Comment:
Changing the enum values is unsafe. We are emitting outbox messages currently.

Reply:
I'll fix this.